src/foam/core/fs/fileDropZone/FileDropZone.js

```diff
@@ -115,7 +115,7 @@ foam.CLASS({
     { name: 'LABEL_MAX_SIZE',      message: 'Max size:' },
     { name: 'ERROR_FILE_TITLE',    message: 'Error' },
     { name: 'ERROR_FILE_TYPE',     message: 'Invalid file type' },
-    { name: 'ERROR_FILE_SIZE',     message: 'File size exceeds 15MB' },
+    { name: 'ERROR_FILE_SIZE',     message: 'File size exceeds' },
```

Comment:
With https://github.com/kgrgreer/foam3/pull/4362
this can now be accomplished with: 
```suggestion
    { name: 'ERROR_FILE_SIZE',     message: 'File size exceeds ${this.maxSize}MB' },
```

Reply:
thank you, will update it then